apps/fabric-website/src/styles/_base.scss

```diff
@@ -124,4 +124,9 @@
       width: 125px;
     }
   }
+
+  /* Hide the YouTube TOC link being shown incorrectly by the shared footer. */
+  [itemtype="http://schema.org/Organization"] a.c-hyperlink {
```

Comment:
Not sure this is specific enough--`document.querySelectorAll('[itemtype="http://schema.org/Organization"]')` returns 4 things. Could we hide it by href instead, or is that type of query too expensive (or too specific to current implementation)?

Reply:
Rather, it appears to be specific enough now (the full query only returns one thing) but I wonder if it might not stay that way.